opm/output/eclipse/RestartIO.cpp

```diff
@@ -437,8 +437,8 @@ namespace {
         writeGroup(sim_step, schedule.getUnits(), schedule, sumState, inteHD, rstFile);
 
         // Write network data if the network option is used and network defined
-        if ((es.runspec().networkDimensions().maxNONodes() >= 1) &&
-            schedule[sim_step].network().active())
+        const auto& network = schedule[sim_step].network();
+        if (network.NoOfNodes() >= 1 && network.active())
```

Comment:
I believe `network.active()` requires both non-zero number of branches and nodes, so there is a slight difference?

Reply:
If you it is  `||`, then it is different.  

Since you are use `&&`,  then the if condition becomes, 
`if ( network.NoOfNodes() >= 1  &&  !this->m_branches.empty() && !this->m_nodes.empty() )` then ` network.NoOfNodes() >= 1 ` is redundant.

Reply:
Ah, you're right - will update.